tests/Linter/ProcessLinterProcessBuilderTest.php

```diff
@@ -61,7 +61,7 @@ public function testPrepareCommandOnPhpOnWindows(string $executable, string $fil
      */
     public static function providePrepareCommandOnPhpOnWindowsCases(): iterable
     {
-        yield 'Linux-like' => ['php', 'foo.php', 'php -l foo.php'];
+        yield 'Linux-like' => ['php', 'foo.php', 'c:\tools\php\php.EXE -l foo.php'];
```

Comment:
are you able to validate on non-CI env if this is good? was a while since I coded on Windows, but I was close to believe it shall be php.exe, not php.EXE

Reply:
I thought it should be `php.exe`, but apparentely not: https://github.com/symfony/process/blob/v7.2.0/Tests/ProcessTest.php#L1703

Reply:
but that's still the CI env (and for some random executable).

I would love to validate it behaves properly for PHP executable on Windows in non-CI env, as everywhere I see php.exe and not php.EXE ([example](https://stackoverflow.com/questions/42136850/execute-php-file-with-php-exe))

Reply:
Maybe we should always use `ExecutableFinder` to normalize path for PHP binary? That should fix the problem for Windows and wouldn't break anything for other OSes.

Reply:
Probably, `c:\tools\php\php.EXE` is definitely wrong, `C:` drive does not even have to exist on a Windows systen.

Reply:
@Wirone , i believe we do already use ExecutableFinder:
https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/master/src/Linter/ProcessLinter.php#L47-L68